modules/t/test-genome-DBs/homo_sapiens/core/meta.txt

```diff
@@ -259,3 +259,4 @@
 1059	\N	patch	patch_112_113_a.sql|schema_version
 1060	\N	patch	patch_112_113_b.sql|Ensure meta_value is not null
 1061	\N	patch	patch_113_114_a.sql|schema_version
+1062    \N	patch   patch_114_115_a.sql|schema_version
```

Comment:
Thanks @jamie-m-a , there are just some final changes for upgrading schema_version to 115 and updating `table.sql` for core and otherfeatures

Reply:
Sorry missed the schema versions...! Updated now. There's no need to increment ISAM for meta, they seem to already be at the correct numbers for core and otherfeatures (I had checked that :) )